packages/run-protocol/test/amm/constantProduct/test-checkInvariants.js

```diff
@@ -119,7 +119,7 @@ function checkGetOutput(t, args, result) {
 
 const testGetInputPrice = (t, inputs, runIn) => {
   const args = runIn ? prepareRUNInTest(inputs) : prepareRUNOutTest(inputs);
-  // @ts-ignore
+  // @ts-ignore typescript doesn't like param list built by destructuring
```

Comment:
I wonder if there's a way to use [`Parameters<Type>`](https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype).

I'm not inspired to work it out just now, though.